lib/modules/accountSwitcher.js

```diff
@@ -434,12 +456,42 @@ function _notifySwitchedAccountElsewhere() {
 		message,
 	});
 }
-
+function autoSwitcher(thing) {
+	if(thing.getAuthor() != loggedInUser() && window.location.search.indexOf('accountSwitcher=auto') == -1 ) {
```

Comment:
DRY: put this at the top of `function autoSwitcher`

`const currentUserNotAuthor === thing.getAuthor() !== loggedInUser();`  and then `if (currentUserNotAuthor && !autoSwitched)`